gammapy/cube/core.py

```diff
@@ -314,30 +312,39 @@ def wcs_pixel_to_skycoord(self, x, y, z):
         energy = Quantity(energy, self.energy_axis.energy.unit)
         return (position, energy)
 
-    def to_sherpa_data3d(self):
+    def to_sherpa_data3d(self, use_intspatialmodel=False):
```

Comment:
Can you introduce a `dstype` argument as used in `SkyImage.to_sherpa_data2d()` (see [here](https://github.com/gammapy/gammapy/blob/master/gammapy/image/core.py#L694)) and add the argument to the docstring?